src/helper/createGarageData.ts

```diff
@@ -13,7 +13,7 @@ export function generateDummyGarages(buildings: BuildingItem[]): GarageItem[] {
           location: `Location ${building.id}-${i + 1}`,
           description: `Description ${building.id}-${i + 1}`,
           usableSpace: 50 * (i + 1),
-          rent: 500 * (i + 1),
+          //rent: 500 * (i + 1),
```

Comment:
I would prefer to leave it as a comment because based on my last discussion with Prof. Stanik, he would like to keep this open for future implementation. Unless @astanik agrees to delete it?